tests/functional/test_user_classes.py

```diff
@@ -56,55 +57,58 @@ def __init__(self, parent, sec):
     model = mm.model_from_str(modelstr)
     # Test that generic First class is created
     assert type(model).__name__ == "First"
-    assert type(model) is not First
+    assert not isinstance(model, First)
 
     mm = metamodel_from_str(grammar, classes=[First, Second])
     model = mm.model_from_str(modelstr)
     # Test that user class is instantiated
     assert type(model).__name__ == "First"
-    assert type(model) is First
+    assert isinstance(model, First)
```

Comment:
what is the reason to prefer `isinstance` here... In many other locations this was not changed... (?) This ok for me. We do no bother with base classes here. ...just a note/question.

Reply:
ruff reports all usages of `type(x) is y`. In most cases it is better to use `isinstance` but in this particular case we don't have to, so we could either change to `isinstance` or use `# noqa` to silence the linter.